tests/tck/features/yield/parameter.feature

```diff
@@ -17,6 +17,23 @@ Feature: Parameter
       | 2    | false | "Tim Duncanef" | 4.1  | [1,true,3] | 3    | true |
 
   Scenario: cypher with parameters
+    When executing query:
+      """
+      MATCH (v) WHERE id(v)==$p3
+      RETURN id(v) AS v
+      """
+    Then the result should be, in any order:
+      | v            |
+      | "Tim Duncan" |
+    When executing query:
+      """
+      MATCH (v) WHERE id(v) IN [$p3,$p8]
+      RETURN id(v) AS v
+      """
+    Then the result should be, in any order:
+      | v               |
+      | "Tim Duncan"    |
+      | "Manu Ginobili" |
```

Comment:
Add test cases:
```
MATCH (v) WHERE id(v) IN $p7.a.b.d RETURN v
```

Reply:
Thanks @czpmango , this case addressed the new case that was not yet supported and now it's fixed.

done.

Reply:
Nice.